CAPI/API/include/Communication.h

```diff
@@ -44,16 +46,17 @@ class Communication
 
     bool TryConnection(int64_t playerID);
     protobuf::MessageToClient GetMessage2Client();
-    bool HaveMessage2Client();
     void AddPlayer(int64_t playerID, THUAI6::PlayerType playerType, THUAI6::HumanType humanType, THUAI6::ButcherType butcherType);
 
     void ReadMessage(int64_t playerID);
 
 private:
     std::unique_ptr<protobuf::AvailableService::Stub> THUAI6Stub;
-    bool haveNewMessage = false;
+    std::atomic_bool haveNewMessage = false;
```

Comment:
```suggestion
    bool haveNewMessage = false;
```
这里不需要用 `atomic`，因为代码中所有对 `haveMessage` 的访问都被 `mutex` 保护，而 `mutex` 的 `locl` 和 `unlock` 已经有内存屏障语义了，不需要再用原子变量来保证一致性。原子变量是用于**无锁编程**的。